src/js/components/HealthChecksComponent.jsx

```diff
@@ -73,7 +74,14 @@ var HealthChecksComponent = React.createClass({
 
     var portIndexClassSet = classNames({
       "col-sm-2": true,
-      "hidden": row.protocol === HealthCheckProtocols.COMMAND
+      "hidden": row.protocol === HealthCheckProtocols.COMMAND ||
+        row.portType === HealthCheckPortTypes.PORT_NUMBER
+    });
+
+    var portClassSet = classNames({
```

Comment:
Yeah, one is the distinguished label caption, the other is the attribute name on the data.